fix typos in html documentations (#1061)
Codecademy/docs: content/html/concepts/videos/videos.md

> - If the video is purely decorative, use `aria-hidden="true"` to denote to screen readers that the video provides no additional context or content and is purely decorative
> - Use the `prefers-reduced-motion` CSS media query in order to disable autoplaying videos for certain users

Reviewer: ```suggestion
- If the video is purely decorative, `aria-hidden="true"` can denote to screen readers that the video provides no additional context or content and is purely decorative.
- The `prefers-reduced-motion` CSS media query can be used to disable autoplaying videos for certain users.
```
